README.md

```diff
@@ -110,41 +110,74 @@ This sample shows how to create custom AnalyzeRules for use within the Analyze w
+
+ **Note**: [Play Asset Delivery](https://docs.unity3d.com/Manual/play-asset-delivery.html) requires Unity 2019.4+
+
+If you want to make [custom asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#custom-asset-packs) use this project as a guide. Otherwise you don't need to change how Addressables is configured in your project. Just follow the instructions specified in the [Play Asset Delivery](https://docs.unity3d.com/Manual/play-asset-delivery.html) manual page.
+
+This is possible because the AddressablesPlayerBuildProcessor moves all content located in Addressables.BuildPath to the streaming assets path. Unity assigns streaming assets to the [generated asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#generated-asset-packs).
+
+##### Basic Workflow
+- Assign an Addressable Group to an asset pack. Multiple groups can be assigned to the same asset pack. Just be mindful of the [size restrictions per delivery mode](https://developer.android.com/guide/playcore/asset-delivery#custom-asset-packs).
+- Unity automatically creates [generated asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#generated-asset-packs) for built-in data and streaming assets. To assign content to the generated asset packs, we can just rely on the default Addressables behavior: any groups that use the Addressables.BuildPath will have their content included in the streaming assets path (see AddressablesPlayerBuildProcessor).
+- For additional [custom asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#custom-asset-packs), an “{asset pack name}.androidpack” directory in the Assets folder must be created for each asset pack, optionally containing a .gradle file specifying the asset pack's delivery type (default type is "on-demand" delivery). See how we do this in BuildScriptPlayerAssetDelivery.cs.
+- To assign an AssetBundle to an asset pack, move the .bundle file to the “{asset pack name}.androidpack” directory (see PlayAssetDeliveryBuildProcessor.cs). We also maintain a system that keeps track of all our custom asset pack names which bundles are assigned to them (see BuildScriptPlayerAssetDelivery.cs and AddressablesInitSingleton.cs).
+- Build the Android App Bundle and upload it to the Google Play Console to generate the APK.
+- At runtime download asset packs before loading bundles from them (see PlayAssetDeliveryAssetBundleProvider.cs).
+
+##### Configure Build & Player Settings
+Configure Unity to build Android App Bundles. For more information see ["Play Asset Delivery"](https://docs.unity3d.com/Manual/play-asset-delivery.html#using-play-asset-delivery).
+1. In File > Build Settings, set the Platform to Android. If "Export Project is enabled", enable "Export for App Bundle". Otherwise, enable "Build App Bundle (Google Play)".
```

Comment:
UI settings should be bolded. In this case, "File", "Build Settings", "Platform", "Android", "Export Project", "Export for App Bundle" and "Build App Bundle (Google Play)" should all be in bold. Don't bold the arrow in between two UI elements i.e. 
Go to **File** > **Build Settings** and set the **Platform** property to **Android**.